curvefs/src/client/inode_cache_manager.cpp

```diff
@@ -118,6 +118,14 @@ CURVEFS_ERROR InodeCacheManagerImpl::GetInode(uint64_t inodeId,
                     << inodeId << " opencount is 0";
             iCache_->Remove(inodeId);
         } else {
+            curve::common::UniqueLock lgGuard = out->GetUniqueLock();
```

Comment:
Refreshing directly on the path of getinode will reduce performance. You can consider getinode triggered and refresh asynchronously.

Reply:
If refresh s3ChunkInfo asynchronously, you need combine the s3ChunkInfo with cached one which may increased.
The combine is not easy and this process will also block io if write at same time.